### PR TITLE
Use OS independent compare for file paths

### PR DESCRIPTION
**Pull request by rkesters**
Please:
- [X] Make your pull request atomic, fixing one issue at a time unless there are many relevant issues that cannot be decoupled.
- [ ] Provide a test case & update the documentation in the `Readme.md`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,7 +51,8 @@
     "safe-stable-stringify": "^2.2.0",
     "ts-node": "^10.2.1",
     "typescript": "~4.5.0",
-    "yargs": "^17.1.1"
+    "yargs": "^17.1.1",
+	 "path-equal": "1.1.2"
```
Comment (domoritz) [inline package.json:55]: Indent correctly

**Review by domoritz — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (rkesters) [reply package.json]: Updated

File: package.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
     "ts-node": "^10.2.1",
     "typescript": "~4.5.0",
     "yargs": "^17.1.1",
-	 "path-equal": "1.1.2"
+	"path-equal": "1.1.2"
```
[the remark this answers is not in the document]
Comment (rkesters) [reply package.json]: Fixed

**Comment by domoritz** [comment]
Thank you!

**Pull request merged**